contracts/transmuter/src/transmuter_pool/weight.rs

```diff
@@ -51,6 +47,14 @@ impl TransmuterPool {
         Ok(Some(ratios))
     }
 
+    pub fn std_norm_factor(&self) -> Result<Uint128, ContractError> {
```

Comment:
Can we add a doc for what this is / whats going on here

I don;t quite understand what lcm from iter/whats happening here.